.storybook/stories/KvToggle.stories.js

```diff
@@ -1,28 +1,157 @@
 import KvToggle from '@/components/Kv/KvToggle';
+import { boolean } from '@storybook/addon-knobs';
 
 export default { title: 'Kv | Form Elements/KvToggle' };
 
 export const Default = () => ({
 	components: {
 		KvToggle
 	},
-	data() {
-		return {
-			checked: true,
-		}
+	props: {
```

Comment:
You can't mutate props directly like this. See console: 

[Vue warn]: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "option3"

Reply:
Good catch, switched to data() to demonstrate the v-model stuff